eclipse-theia/theia
# 4016 debug vscode ext

#### What it does
fixed #4016 with these functionality:
1. Supports start, restart, stop of pwa-extensionHost launch on debug toolbar.
2. Doesn't support stepping in/out, pause in debug toolbar.

#### How to test
1. Add pwa-extensionHost configuration to launch configurations like so:
![image](https://user-images.githubusercontent.com/71069170/97988704-0cd68100-1de6-11eb-85ec-7dc9c456185f.png)
2. start debugging from debug toolbar
3. two threads will eventually appear after the hosted plugin windows will be opened - like so:
![image](https://user-images.githubusercontent.com/71069170/97988834-40b1a680-1de6-11eb-93cf-ee18312caeee.png)
4. you can choose restart or stop in debug toolbar.

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
